### DISCUSSION TITLE
website fyne-io(apps) with donations

### DISCUSSION THREAD

**Discussion by ghost** [fyne-io/fyne#3285] [Q&A] [answered]
Hi everyone.

I had a random idea today. *What if the [apps-fyne-io](https://apps.fyne.io/) website had a payment link to encourage your development?*

#### Idea
![image](https://user-images.githubusercontent.com/111701513/192251586-e9f83c66-f1f0-4423-8f88-d753e561bb4a.png)

What do you all think of this idea? 

<br/>

**Why?** There are things today like **gnome-store** - my idea would be something like **gnome-store** for fyne-io.

**Comment by Jacalz** [comment, marked as answer]
I think this makes more sense over at https://github.com/fyne-io/apps.fyne.io. This discussion space is meant mostly for the Fyne toolkit itself. The website parts are divided up into their own repositories.

**Reply by ghost** [reply]
Hi Jacalz. thank you for feedback.
